includes/wikia/resourceloader/ResourceLoaderGlobalWikiModule.class.php

```diff
@@ -99,22 +102,22 @@ protected function getContent( $title, $titleText, $options = array() ) {
 			*/
 
 			if ( WikiFactory::isWikiPrivate( $title->getCityId() ) == false ) {
-				$content = $title->getContent();
+				if ( !is_null( $revisionId ) ) {
```

Comment:
It would be great to see few lines of explanation of purpose of this block in comment.

this condition is strictly connected to content review and this relation is hard to discover here